recipes/ace/bld.bat

```diff
@@ -0,0 +1,67 @@
+REM Installation following the instructions in 
+REM https://htmlpreview.github.io/?https://github.com/DOCGroup/ACE_TAO/blob/master/ACE/ACE-INSTALL.html
+set ACE_ROOT=%SRC_DIR%
+set ACE_SOURCE_PATH=%ACE_ROOT%\ace
+set WORKSPACE=%ACE_ROOT%\ace\ace
+set SLN_FILE=%WORKSPACE%.sln
+set SLN_CFG=Release
+set SLN_PLAT=x64
+
+echo %SRC_DIR%
+echo %ACE_ROOT%
+echo %PREFIX%
+echo %LIBRARY_INC%
+echo %SLN_PLAT%
+
+REM Configure step
+cd %ACE_ROOT%
+perl %ACE_ROOT%\bin\mwc.pl -type vs2017 -features "uses_wchar=1,zlib=0,ssl=0,openssl11=0,trio=0,xt=0,fl=0,fox=0,tk=0,qt=0,rapi=0,stlport=0,rwho=0" %WORKSPACE%.mwc
```

Comment:
What about vs2019?

Reply:
we only have vs2017 on conda-forge afaik.